### ISSUE TITLE
Ignore files in Source Control View based on exclude patterns defined in files.exclude setting

### ISSUE THREAD

**Issue by sexybiggetje** [JohnstonCode/svn-scm#102]
- VSCode Version: 1.20.0-inside
- OS Version: Windows 10.0.150.63
- Extension Version: 1.5.0
- System Language: nl_NL (Dutch, Nederlands)

In my settings file, I've configured files.exclude and search.exclude to ignore certain patterns. These matching files and folders however are not ignored in the Source Control View. For example, at the office we have /dist/ and /bower_components/ in Subversion, I choose to ignore those locally since I don't do the building. Adding them to svn:ignore isn't an option for me.

TL:DR; Please add the possability to ignore certain files/folders either by adhering to the files.exclude setting or a custom setting.

**Comment by JohnstonCode** [comment]
@sexybiggetje 

Thanks for creating an issue. This is something that could be added to the extension settings.

**Comment by JohnstonCode** [comment]
This has been added in 1.6.0

edit the "svn.multipleFolders.ignore" setting.

**Comment by sexybiggetje** [comment]
Thanks, works great!
